frontend/ui/data/onetime_migration.lua

```diff
@@ -760,5 +760,13 @@ if last_migration_date < 20241123 then
     end
 end
 
+-- 20241207, Move PatchManagement plugin to the core
+-- https://github.com/koreader/koreader/pull/12862
+if last_migration_date < 20241207 then
+    logger.info("Performing one-time migration for 20241207")
+
+    ffiUtil.purgeDir(DataStorage:getDataDir() .. "/plugins/patchmanagement.koplugin")
```

Comment:
The wording of the comment should probably read as:
`-- We moved patch management to core. Remove the original plugin.`

(This migration step does _not_ "Move PatchManagement plugin to the core", _you_ did :))